tools/sgyamlfmt/tools.go

```diff
@@ -12,23 +14,51 @@ import (
 	"go.einride.tech/sage/sgtool"
 )
 
+//go:embed .yamlfmt.yaml
+var DefaultConfig []byte
+
 const (
-	version    = "0.2.9"
-	binaryName = "yamlfmt"
+	name              = "yamlfmt"
+	version           = "0.5.0"
+	defaultConfigName = ".yamlfmt.yaml"
 )
 
 func Command(ctx context.Context, args ...string) *exec.Cmd {
 	sg.Deps(ctx, PrepareCommand)
-	return sg.Command(ctx, sg.FromBinDir("yamlfmt"), args...)
+	return sg.Command(ctx, sg.FromBinDir(name), args...)
+}
+
+// Run, runs the google/yamlfmt tool.
+// If no config file is found in the git root then it will run with the default config.
+func Run(ctx context.Context, args ...string) error {
+	defaultConfigPath := sg.FromToolsDir(name, defaultConfigName)
```

Comment:
You can still write the file using the default name '.yamlfmt'

Reply:
Ah, understood what ur getting at. Keep a "sane" file name here, whilst writing it as `yamlfmt` expects it.

Reply:
Exactly!